src/modules/ModuleStaking/index.vue

```diff
@@ -154,32 +163,33 @@ function handleStakedUnstaked() {
     </template>
     <div
       v-if="isLoading && !showViewMore"
-      v-bem="'loader'"
+      :class="$style.loader"
+      class="flex-1 flex justify-center items-center"
     >
       <KlayLoader />
     </div>
   </div>
 </template>
 
-<style lang="sass">
-$padding-bottom: 19px
-
-.module-staking
-  flex: 1
-  display: flex
-  flex-direction: column
-  padding-bottom: $padding-bottom
-  &__view-more
-    display: flex
-    justify-content: center
-    width: 100%
-    padding: 8px 0
-    margin-bottom: - $padding-bottom
-  &__loader
-    flex: 1
-    display: flex
-    justify-content: center
-    align-items: center
-    min-height: 82px + $padding-bottom
-    margin-bottom: - $padding-bottom
+<style lang="scss" module>
+@use '@/styles/vars';
+
+$padding-bottom: 19px;
+
+.module-staking {
+  padding-bottom: $padding-bottom;
+}
+
+.view-more {
+  margin-bottom: -$padding-bottom;
+}
+
+.loader {
+  min-height: 82px + $padding-bottom;
+  margin-bottom: -$padding-bottom;
+}
+
+.empty {
+  color: vars.$gray3;
+}
 </style>
```

Comment:
Looks like you're repeating yourself. Maybe, move that CSS module into a separate file?

Reply:
Yes, I agree. I've been thinking about it. I think to put it in a separate component. I'll do it in another PR